[2025-04-22T22:29:55.482+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:29:36.102757+00:00 [queued]>
[2025-04-22T22:29:55.487+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:29:36.102757+00:00 [queued]>
[2025-04-22T22:29:55.488+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-04-22T22:29:55.495+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): meteo_operations.save_meteo_data> on 2025-04-22 22:29:36.102757+00:00
[2025-04-22T22:29:55.500+0000] {standard_task_runner.py:57} INFO - Started process 1243 to run task
[2025-04-22T22:29:55.503+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.save_meteo_data', 'manual__2025-04-22T22:29:36.102757+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp064wgesl']
[2025-04-22T22:29:55.504+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask meteo_operations.save_meteo_data
[2025-04-22T22:29:55.534+0000] {task_command.py:415} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.save_meteo_data manual__2025-04-22T22:29:36.102757+00:00 [running]> on host ba2ad926eb92
[2025-04-22T22:29:55.585+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.save_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-22T22:29:36.102757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-22T22:29:36.102757+00:00'
[2025-04-22T22:29:55.599+0000] {python.py:194} INFO - Done. Returned value was: Successfully inserted measurement into database
[2025-04-22T22:29:55.612+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.save_meteo_data, execution_date=20250422T222936, start_date=20250422T222955, end_date=20250422T222955
[2025-04-22T22:29:55.635+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-22T22:29:55.650+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
